app/controllers/games_controller.rb

```diff
@@ -0,0 +1,29 @@
+class GamesController < ApplicationController
+
```

Comment:
Careful on spacing. Set up your editor to use 2 space tabs. IE no tabs and a tab auto converts to 2 spaces. Then respace this document.